models/endpoint-model.js

```diff
@@ -0,0 +1,8 @@
+const fs = require("fs/promises");
+const path = "./endpoints.json";
+
+exports.readEndpoints = () => {
```

Comment:
As you aren't dealing with the database, this logic should be in the controller

Reply:
As not to get confused I am keeping it separate in model.